tests/lava/magma/runtime/message_infrastructure/test_all_delivery.py

```diff
@@ -12,8 +12,8 @@
 from multiprocessing import Process
 from lava.magma.runtime.message_infrastructure.multiprocessing \
```

Comment:
put this import into calling func. we need to confirm all pass at mac & windows.